developer/frameworks/articles/09-info-framework/01-the-info-framework-intro.markdown

```diff
@@ -0,0 +1,81 @@
+# The Info Framework
+
+The Info Framework provides enhances flexibility for retrieving, processing, 
+and displaying any information, regardless of where the information comes from. 
+The Info Framework can process data from the database, created 
```

Comment:
This sentence is not really correct. The Info Framework does not process data. I know it's hard to explain because it can (potentially) do many things with data. That's why I carefully chose the wording of the first paragraph of the draft. Please check it again.

Reply:
Other than the typo noted above, this was an attempt to condense the first couple sentences from your draft. Is the issue here that you're seeing a difference in connotation between "data" and "any type of information"/"information" as you use it in your draft?

Reply:
Not sure, but I feel like my paragraph is correct and this one is inaccurate and a bit confusing. I'd rather have a longer paragraph that is more correct.